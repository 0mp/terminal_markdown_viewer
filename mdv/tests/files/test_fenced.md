fenced code

```
class C1(object):
    pass
```
