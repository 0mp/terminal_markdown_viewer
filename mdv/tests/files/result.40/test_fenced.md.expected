[38;5;188m  
  [38;5;59m| [0m [38;5;102m[38;5;74mclass[0m [38;5;209mC1[0m([38;5;209mobject[0m): pass
[0m[0m


